CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Accessing App::current_user() from within a notification produced by App:switch_user() (which includes notifications for a newly logged in user) would deadlock ([#7670](https://github.com/realm/realm-core/issues/7670), since v14.6.0).
 * Inserting the same typed link to the same key in a dictionary more than once would incorrectly create multiple backlinks to the object. This did not appear to cause any crashes later, but would have affecting explicit backlink count queries (eg: `...@links.@count`) and possibly notifications ([#7676](https://github.com/realm/realm-core/issues/7676) since v14.5.2).
 * Automatic client reset recovery would crash when recovering AddInteger instructions on a Mixed property if its type was changed to non-integer ([PR #7683](https://github.com/realm/realm-core/pull/7683), since v11.16.0).
+* Avoid deadlock when opening a new Realm requiring schema update while an async transaction is in progress ([#7693](https://github.com/realm/realm-core/issues/7693), since v11.8.0)
```

Comment:
we technically avoid it, but the app may crash. maybe that should be mentioned as a way of "preventing" the deadlock